woocommerce/woocommerce-google-analytics-integration
# Fix - Selectors in Product Blocks

### Changes proposed in this Pull Request:



Closes #377 and #381 

Handle the `add_to_cart` and `select_content` events for the block in the same way we do for classic pages. Add listeners to the component internals, as the Experimental Hooks stopped working.


### Detailed test instructions:


1. Install Store Front
2. GO to shop page
   0. Verify `view_item_list` is tracked for all the products on the page
   1. `add_to_cart` is tracked when you click on "Add to cart" button
   4.  `select_content` & no `add_to_cart` is tracked, when
       1. Click on product image
       2. Click on the product name
       3. Click on "Select options"
       5. Click on "View products"

6. Create a Page and add Products Block (Beta) 
  Repeat and verify steps from step 2 


### Additional details:


### Changelog entry

> Fix - Add to cart & select content events for All Products Block.
